Jenkinsfile

```diff
@@ -388,8 +388,18 @@ pipeline {
         always {
           junit testResults: '**/target/surefire-reports/*.xml'
           container('maven') {
-            // clean up the redis namespace
-            sh "kubectl delete namespace ${NAMESPACE_REDIS} --ignore-not-found=true"
+            script {
+              try {
+                 // clean up the redis namespace
+                // workaround for Cannot delete namespace still in Terminating #NXP-3228
+                sh """
+                  kubectl get namespace ${NAMESPACE_REDIS} -o json | sed -e s/\\"kubernetes\\"//g  | kubectl replace --raw /api/v1/namespaces/${NAMESPACE_REDIS}/finalize -f -
+                  kubectl delete namespace ${NAMESPACE_REDIS} --ignore-not-found=true --cascade=true --force=1 --grace-period=0
+                """
+              } catch (err) {
```

Comment:
I'm not in favor of catching any error here: if there is an error in the workaround or the deletion, it is not expected, so we want to be aware of it even if it makes the build fail.

This is how we detected the namespace deletion problem in the first place, because the jobs were red.

Reply:
we can remove the try catch and monitor the namespace  deletion 
if many build failed due to this error perhaps we should reactivates the try catch.

Reply:
yep, I think the workaround will do its job fine